### PR TITLE
Fix build script for cekit

### LINKED ISSUE #399: cekit build needs a refresh
cekit build seems broken.
no need to know the operator's code details to fix it, you just need:
- cekit experience
- golang build tools experience

cekit stuff is here:
./build/cekit

### PR DESCRIPTION
**Pull request by dmvolod**
Fixes #399

### CODE REVIEW

File: build/cekit/modules/builder-infinispan-operator/install [hunk 1]
```diff
@@ -4,35 +4,26 @@ set -x -e
...
-#go test ./...
-CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -v -a -o ${OPERATOR_PATH} github.com/infinispan/infinispan-operator/cmd/manager
+go mod vendor
+CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o ${OPERATOR_PATH} ${BUILD_PATH}
 if [[ ! -f ${OPERATOR_PATH} ]] ; then
     echo "${OPERATOR_PATH} does not exist, aborting."
     exit 1
 fi
 
-#mv build/_output/bin/* /usr/local/bin/
```
Comment (dmvolod) [inline build/cekit/modules/builder-infinispan-operator/install:30]: @rigazilla do you know, should we keep all commented lines or not?

Comment (rigazilla) [reply build/cekit/modules/builder-infinispan-operator/install:30]: @dmvolod I think we can cleanup, thanks!

Comment (dmvolod) [reply build/cekit/modules/builder-infinispan-operator/install:30]: Thanks for response, @rigazilla . 
Looks like ${INSTALL_PKGS} and go-toolset are RHEL specific.

Comment (rigazilla) [reply build/cekit/modules/builder-infinispan-operator/install:30]: @dmvolod it's ok to remove comments by me. Let's stay with Fedora upstream

Comment (dmvolod) [reply build/cekit/modules/builder-infinispan-operator/install:30]: @rigazilla , fixed, please review

Comment (rigazilla) [reply build/cekit/modules/builder-infinispan-operator/install:30]: @dmvolod looks great to me! please squash the commits, I'll merge then. Thanks!

Comment (dmvolod) [reply build/cekit/modules/builder-infinispan-operator/install:30]: Thanks, @rigazilla , done.

**Comment by rigazilla** [comment]
merged, thanks!

**Pull request merged**